### DISCUSSION TITLE
如何移除recent posts挂件的categories

### DISCUSSION THREAD

**Discussion by declan-haojin** [ppoffice/hexo-theme-icarus#1112] [Q&A] [answered]
如图请问如何移除recent posts挂件的categories，我在/layout下没有找到对应的代码。因为如果类别名字过长，排版会不太美观。或者请问可以设置只显示第一级分类吗？谢谢！

---

<img width="295" alt="image" src="https://user-images.githubusercontent.com/56029057/187123402-a4beb620-6421-4522-99cc-23ec4718008a.png">

**Comment by ppoffice** [comment, marked as answer]
@declan-haojin https://github.com/ppoffice/hexo-component-inferno/blob/2.0.1/src/view/widget/recent_posts.jsx#L85-L88
